### PR TITLE
Clean up Razor EA

### PR DESCRIPTION
**Pull request by davidwengier**
Now that Razor is on dynamic registration, we don't need our own server for cohosting.

Part of https://github.com/dotnet/razor/issues/9519

### CODE REVIEW

File: src/Tools/ExternalAccess/Razor/Cohost/AbstractRazorLspService.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.CodeAnalysis.ExternalAccess.Razor.Cohost;
 
 /// <summary>
-/// Base class for services that need to live in Razor but be exported using <see cref="ExportRazorStatelessLspServiceAttribute"/>
+/// Base class for services that need to live in Razor cannot but be exported using <see cref="ExportCohostStatelessLspServiceAttribute"/>
```
Comment (ady109) [inline src/Tools/ExternalAccess/Razor/Cohost/AbstractRazorLspService.cs:10]:
```suggestion
/// Base class for services that need to live in Razor but cannot be exported using <see cref="ExportCohostStatelessLspServiceAttribute"/>
```

Comment (davidwengier) [reply src/Tools/ExternalAccess/Razor/Cohost/AbstractRazorLspService.cs]: lol, whoops. Thanks!

**Review by dibarbet — APPROVED** [review]

**Review by alexgav — APPROVED** [review]
:shipit:

**Review by DustinCampbell — APPROVED** [review]

**Comment by davidwengier** [comment]
Woohoo, images were updated, I can merge this!

**Pull request merged**